kivy/core/window/window_sdl2.py

```diff
@@ -15,21 +15,22 @@
 
 __all__ = ('WindowSDL2', )
 
-from os.path import join
 import sys
+from collections import deque
+from os.path import join
+
 from kivy import kivy_data_dir
-from kivy.logger import Logger
 from kivy.base import EventLoop, ExceptionManager, stopTouchApp
 from kivy.clock import Clock
+from kivy.compat import unichr
 from kivy.config import Config
 from kivy.core.window import WindowBase
 from kivy.core.window._window_sdl2 import _WindowSDL2Storage
-from kivy.input.provider import MotionEventProvider
 from kivy.input.motionevent import MotionEvent
+from kivy.input.provider import MotionEventProvider
+from kivy.logger import Logger
 from kivy.resources import resource_find
 from kivy.utils import platform, deprecated
-from kivy.compat import unichr
-from collections import deque
```

Comment:
Why the sudden shifting of lines? It's not like we need them in that specific order(please revert the change). 😋